registry.json

```diff
@@ -3600,5 +3600,19 @@
     "telegram": "SmaugPool",
     "operatorName": "",
     "github": "SmaugPool"
+  },
+  {
+    "ticker": "KAKIS",
+    "poolId": "09ae473a0781eefa6345e2236a8ffa92c26dc151c93a8f67ae735860",
+    "social": {
+      "website": "https://ada.stakingkakis.com",
+      "twitter": "stakingkakis",
```

Comment:
Please could you add some activity to your social media? There are no follower, no following and only one tweet...